docs/quickstart/installation.md

```diff
@@ -77,9 +77,14 @@ pip install torch==1.13.1 torchvision functorch --extra-index-url https://downlo
 :::
 ::::
 
+### ninja
+```bash
+pip install ninja
+```
+
 ### tinycudann
 
-After pytorch, install the torch bindings for tinycudann:
+After pytorch and ninja, install the torch bindings for tinycudann:
 
 ```bash
 pip install git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch
```

Comment:
It might be nicer to update this command to
`pip install ninja git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch`

Reply:
Yeah! And I also notice that ninja is required when installing nerfstudio so it might be nicer to install it first. I will update it.